src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container container-big">
+      <form
+        action=" "
+        method="post"
+        data-qa="big keypress"
+      >
+        <img
+          class="image"
+          alt="search"
+          src="C:\nadiia-rusnak\projects\layout_search-bar-airbnb\src\src\images\Search.svg"
```

Comment:
you should set relative path here for image (corresponding to the index.html file), not the absolute one